app/modules/cdr/endpoints_cdr.py

```diff
@@ -2760,14 +2760,109 @@ async def scan_ticket(
             db=db,
             ticket_id=ticket.id,
             scan=ticket.scan_left - 1,
-            tags=ticket.tags + "," + ticket_data.tag.strip(),
+            tags=ticket.tags + "," + ticket_data.tag.strip()
```

Comment:
If you need to strip, you could do it with a validator in the model, like the lowercase step